runtimes/runtimes/auth/auth.ts

```diff
@@ -71,6 +75,9 @@ export class Auth {
                 if (type === 'bearer') {
                     return this.bearerCredentials !== undefined
                 }
+                if (type === 'bearer-alternate') {
```

Comment:
instead of overloading the existing credentials provider, add a new getAtxCredentialsProvider() or similar so it's clear what the purpose of this alternate credential is